src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -179,7 +179,7 @@ public override async ValueTask<FormulaValue> Visit(CallNode node, SymbolContext
             {
                 if (FuncsByName.TryGetValue(func, out var ptr))
                 {
-                    var result = await ptr(this, childContext, node.IRContext, args);
+                    var result = await ptr(this, new EvalVisitorContext(childContext, context.StackDepthCounter).IncrementStackDepthCounter(), node.IRContext, args);
```

Comment:
>new EvalVisitorContext(childContext, context.StackDepthCounter).IncrementStackDepthCounter() [](http://example.com/codeflow?start=49&length=92)

nit: consider adding a overload IncrementStackDepthCounter so we can just write: 

                    var result = await ptr(this, context.IncrementStackDepthCounter(childContext), node.IRContext, args);

Reply:
Done.